tbx/static_src/sass/components/_streamfield.scss

```diff
@@ -0,0 +1,62 @@
+@use "config" as *;
+
+.streamfield {
+    &__title {
+        margin-bottom: $spacer-small;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            grid-column: 4 / span 9;
+        }
+    }
+
+    &__intro {
+        margin-bottom: $spacer-medium;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            margin-bottom: $spacer-large;
+            grid-column: 4 / span 7;
+        }
+    }
+
+    &__heading {
+        margin-bottom: $spacer-mini;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            grid-column: 4 / span 7;
+        }
+    }
+
+    &__paragraph {
+        @include font-size(body);
+        margin-bottom: $spacer-medium;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            margin-bottom: $spacer-large;
+            grid-column: 4 / span 7;
+        }
+    }
+
+    &__image {
+        margin-bottom: $spacer-medium;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            margin-bottom: $spacer-large;
+            grid-column: 4 / span 7;
+        }
+    }
+
+    &__quote {
+        margin-bottom: $spacer-medium;
+        grid-column: 2 / span 4;
+
+        @include media-query(large) {
+            margin-bottom: $spacer-large;
+            grid-column: 4 / span 9;
+        }
+    }
```

Comment:
It would be good to also set the columns for the raw_html and markdown blocks here - have a look at http://localhost:8000/admin/pages/1814/edit/ which uses a raw_html block. I'm unsure if we should add video - Anna wanted to remove it but the back-end code hasn't been removed. Might be worth raising at stand-up.

Reply:
Aligned raw html and markdown here: https://github.com/torchbox/torchbox.com/pull/33/commits/1bf733aaef8267a716fa59019376fc9ef4343d43